package.json

```diff
@@ -20,8 +20,7 @@
   "scripts": {
     "start": "yarn storybook",
     "build": "yarn copy && ttsc",
-    "build:truss": "cd ./truss && npm run generate",
-    "watch:truss": "cd ./truss && watch 'npm run generate' ./",
+    "build:truss": "truss",
```

Comment:
This is great! So we just need to call truss with a file named `truss-config.ts` and it works!?

Reply:
@KoltonG yep